content/pages/home/content.json

```diff
@@ -1,23 +1,39 @@
 {
   "teaser": {
+    "caption": "Energy SSH",
     "title": "Challenges in mobility",
     "text": "The mobility sector is facing several challenges: from managing CO2 emissions and the transition to electric vehicles and alternatives modes of mobility to maintaining road security, reducing traffic jams, parking problems, and ensuring inclusivity.\n\nThe Future Mobility Data Marketplace matches supply and demand of any kind of data and algorithms that can be relevant for mobility. It allows data scientists, data consumers and data providers, like cities, mobility service providers and automotive companies to share, collaborate and commercialize their data services. In a secure and privacy friendly manner and ensuring that data owners stay in full control over their data\n\nThe Future Mobility Data Marketplace is based on the Gaia-X concepts. Gaia-X is part of the “Digital Europe” program aiming at making Europe's digital infrastructure future proof and self-sovereign.\n\n- Digital Sovereignty - Data owner stays in control\n- Preventing copies of data - Data stays at the source\n- Bridging supply and demand of data and AI algorithms\n- Sharing options for both open and restricted data\n- Labels - “Digital quality stamps” for datasets and algorithms"
   },
-  "points": [
-    { "text": "Managing CO2 emissions" },
-    { "text": "Transition to Electric Vehicles" },
-    { "text": "Move to Mobility as a Service" },
-    { "text": "Maintaining road security" },
-    { "text": "Reducing traffic jams" },
-    { "text": "Solving parking problems" },
-    { "text": "Ensuring inclusivity" },
-    { "text": "Preventing platform monopolies" },
-    { "text": "Citizen privacy" },
-    { "text": "Sustainable business models" }
-  ],
+  "points": {
+    "caption": "Who is EnergySHR",
+    "list": [
+      {
+        "text": "Federated data",
+        "subtext": "data remains at the source"
+      },
+      {
+        "text": "Federated analytics",
+        "subtext": "secure computation over linked datasets"
+      },
+      {
+        "text": "Verifiable data and algorithms",
+        "subtext": "digital quality and compliance certificates"
+      },
+      {
+        "text": "Digital sovereignty",
+        "subtext": "data owners stay in control"
+      }
+    ]
+  },
+  "getInvolved": {
+    "title": "Get involved",
+    "text": "Lorem ipsum dolor sit amet consectetur. Non vel risus integer aliquam. Parturient ac vulputate.",
```

Comment:
we need the correct text here?

Reply:
We don't have a correct text yet. The text used is from FMDM porta. The last update from Maria is that she received the text from the client but she needs to clean it up.

Reply:
should that not be implemented in this ticket?

Reply:
Will be handled in this ticket: [VDX-239](https://sphereon.atlassian.net/browse/VDX-239)